### DISCUSSION TITLE
RocketController relay seems dead but?

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#16611] [Q&A] [answered]
I bought one of these on aliexpress a couple of months ago.
https://rocketcontroller.com/product/controller-esp32r4/
The seller, RobotDyn Official Store was nice enough but only until I started having problems.

The device was working ok until the other day when the buttons stopped working.
I restarted the device and it came back with wifi but nothing else worked. 
I decided to try updating the firmware to the current tasmota but that didn't change anything.
All seemed to be working but the template was somehow lost.

The seller started telling me to do very basic things like re-flashing etc and I kept telling him that I understood but the device was no longer working properly. He sent me a template which I used but still nothing. The hardware buttons can control the relays but they are very slow to respond and erratic.

Then he suggested I use a .dmp file;
Config_tasmota_F747B8_1976_11.0.0.7_RF433_I2C1R8.dmp

Once I did that, it stopped completely. It no longer responds to anything. It is no longer using wifi and it is not offering an AP.
He suggested I try serial which I did but nothing, no output what so ever.

He insists there is nothing wrong with the device and when I asked for a replacement, even offering to buy more if he replaced it, he blocked me from communicating with him on aliexpress.

I could care less about the device, it's just $20.00 but I'm very curious to know if it's actually dead or not.
Why does he insist there is nothing wrong with this device? I followed all his suggestions and it only made it worse yet he still insisted there is nothing wrong with the device before blocking me.

Since I cannot even get a serial response, I have to assume this device is completely dead but it does seem to boot and still turns on the two LED's it did before becoming inaccessible. 

Is it worth spending any more time on this? Am I missing something small?

**Comment by barbudor** [comment, marked as answer]
By uploading his dmp file to the device, you flashed the vendor's own settings including wifi
So I'm not surprised that it doesn't connect to your wifi nor start a wifi AP

If you can't get serial to work, or even doesn't see any of the basic ESP32 boot message it's either something is dead in the circuit between the ESP32 and your serial adapter, or the ESP itself

Deciding if it's worth to solve that depends all on you. Personally I would try to even if I have to replace the ESP32 module

**Reply by dac2020** [reply]
It's odd because the board seems to have some life but I cannot tell what since I cannot get to serial.

I don't have the tools to deal with chips so I guess I have to give up then. I tested my usb/ttl device and it works fine. 
The relay board is dead and the seller refuses to replace it. 

I cannot tell if the seller was actually rocketcontroller direct or some reseller.
